students/mia/users/views.py

```diff
@@ -1,33 +1,37 @@
-# from django.shortcuts import render
-
 # Create your views here.
+
 import json
-import re 
+import re
+import bcrypt 
 
 from django.http import JsonResponse
 from django.views import View
 
 from .models import User
 
 class SignUpView(View):
-    def post(self, request):
-        try:
+    def post(self, request): 
+        try                : 
```

Comment:
다른 의미의 블록이라 `:` 기준으로 정렬하실 필요는 없습니다!
```suggestion
        try: 
```